Engine/include/math/VectorMath.hpp

```diff
@@ -0,0 +1,189 @@
+//
+// VectorMath.hpp
+// VGD Engine
+//
+// Created by Benjamin Arnold on 29 Aug 2015
+//
+
+/*! \file VectorMath.hpp
+* @brief defines common vector math operations.
+*/
+
+#pragma once
+
+namespace vgd {
```

Comment:
That's why I was saying that we should make it more unique. If you're confident that it's not important right now, then we can leave it how it is.